contracts/foundry.toml

```diff
@@ -10,3 +10,16 @@ ffi = true
 ast = true
 build_info = true
 extra_output = ["storageLayout"]
+evm_version = 'shanghai' # Change as needed
```

Comment:
I'd suggest not committing this config to the repo as it's specific to an outdated version of forge